backend/src/yeomanui.ts

```diff
@@ -53,9 +53,7 @@ export class YeomanUI {
 
   constructor(rpc: IRpc, youiEvents: YouiEvents, outputChannel: YouiLog, logger: IChildLogger, uiOptions: any, outputPath: string = YeomanUI.PROJECTS) {
     this.rpc = rpc;
-    if (!this.rpc) {
-      throw new Error("rpc must be set");
-    }
```

Comment:
why it is not needed?

Reply:
rpc is always passed, no need to make this check